services/httpoverrpc/client/client.go

```diff
@@ -192,18 +194,22 @@ type getCmd struct {
 	headers             repeatedString
 	body                string
 	showResponseHeaders bool
+	protocol            string
+	hostname            string
 }
 
 func (*getCmd) Name() string     { return "get" }
 func (*getCmd) Synopsis() string { return "Makes a HTTP call to a port on a remote host" }
 func (*getCmd) Usage() string {
-	return `get [-method METHOD] [-header Header...] [-body body] remoteport request_uri:
-    Make a HTTP request to a specified port on the remote host, defaulting to GET.
+	return `get [-method METHOD] [-header Header...] [-body body] [-protocol Protocol] [-hostname Hostname] remoteport request_uri:
+    Make a HTTP request to a specified port on the remote host.
 `
 }
 
 func (g *getCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&g.method, "method", "GET", "Method to use in the HTTP request")
+	f.StringVar(&g.protocol, "protocol", "http", "protocol to communicate with specified hostname")
+	f.StringVar(&g.hostname, "hostname", "localhost", "ip address or domain name to specify host")
```

Comment:
With https://github.com/Snowflake-Labs/sansshell/pull/283, we need to enable `--allow-any-host` flag, so let's add a note here